tools/checklocks/checklocks.go

```diff
@@ -167,17 +169,15 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			continue
 		}
 
-		var start time.Time
```

Comment:
I see. This is the start of the function. not the package. Should probably revert this suggestion. Apologies for the misunderstanding.